pkg/irqs/info_test.go

```diff
@@ -57,12 +57,35 @@ func TestReadStats(t *testing.T) {
 func validCounters(counters irqs.Stats) bool {
 	// check fakeInterrupts for the magic values
 	// we check some random values atm, not all the expected values
-	if counters[3]["12"] != 713 {
+	if counters[0]["131"] != 3949116 {
```

Comment:
neat: add
```validate(n1, n2) bool {
        return n1 == n2
}
res := validate(counters[0][\"131\"], 3949116)
res == res && validate```

just saves some code lines

Reply:
maybe I can just inline this function then. I'll evaluate this option.